src/dataset-builder/CohortEditor.ts

```diff
@@ -10,19 +10,17 @@ import { BuilderPageHeader } from 'src/dataset-builder/DatasetBuilderHeader';
 import {
   AnyCriteria,
```

Comment:
I'm struggling to understand the real meaning of these changes. I'm wondering if you can add a description/diagram/something about the motivation and impact of these changes.